web/src/components/core/InstallationFinished.jsx

```diff
@@ -46,24 +46,22 @@ function InstallationFinished() {
         </Button>
       </MainActions>
 
-      <Center>
-        <EmptyState>
-          <EmptyStateIcon icon={() => <Icon name="check_circle" />} className="success-icon" />
-          <Title headingLevel="h2" size="4xl">
-            Congratulations!
-          </Title>
-          <EmptyStateBody className="pf-c-content">
-            <div>
-              <Text>The installation on your machine is complete.</Text>
-              <Text>
-                At this point you can 'Reboot' the machine to log in to the new system.
-              </Text>
-              <Text>Have a lot of fun! Your openSUSE Development Team.</Text>
-            </div>
-          </EmptyStateBody>
-        </EmptyState>
-      </Center>
-    </>
+      <EmptyState>
+        <EmptyStateIcon icon={({ ...props }) => <Icon name="check_circle" { ...props } />} className="color-success" />
+        <Title headingLevel="h2" size="4xl">
+          Congratulations!
+        </Title>
+        <EmptyStateBody className="pf-c-content">
+          <div>
+            <Text>The installation on your machine is complete.</Text>
+            <Text>
+              At this point you can 'Reboot' the machine to log in to the new system.
+            </Text>
+            <Text>Have a lot of fun! Your openSUSE Development Team.</Text>
```

Comment:
Just a thought: we might consider replacing the "openSUSE Development Team" (it is not only about installing openSUSE).

Reply:
I agree. Let's think about it and sent a PR with the new text.